### ISSUE TITLE
Please add a feature so that we can insert our own icons for the slider widgets.

### ISSUE THREAD

**Issue by 79ho3ein** [elementor/elementor#14976]
Please add a feature so that we can insert our own icons for the slider widgets. https://prnt.sc/131yadk
Unfortunately we are not able to change the default icons at this time.
This is a small but very practical application. Now the slider tool is very simple. I hope you add more features to this widget soon.

thanks

**Comment by black-eye** [comment]
+1

**Comment by fabianenzensperger** [comment]
It is like my request #14140 … Overall more options for all widgets with Icons would be awesome. 👀

**Comment by 79ho3ein** [comment]
@fabianenzensperger 
Yes, exactly. For example, the list widget in Mentor does not have the ability to insert icons, and this is a big weakness.

**Comment by 79ho3ein** [comment]
How long does it take to add such simple features to an element?
